content/en/docs/configuration/acme/dns01/_index.md

```diff
@@ -80,6 +80,15 @@ with `--set`:
 --set 'extraArgs={--dns01-recursive-nameservers-only,--dns01-recursive-nameservers=8.8.8.8:53\,1.1.1.1:53}'
 ```
 
+If you are deploying using kubectl, then you can add the following line, to the `spec.template.spec.containers.args`
+```bash
+- args:
+        - --v=2
+        - --cluster-resource-namespace=$(POD_NAMESPACE)
+        - --leader-election-namespace=kube-system
+        - --dns01-recursive-nameservers="8.8.8.8:53"
+```
```

Comment:
We're in the process of adding a config file mechanism, which will allow these settings to be configured using a ConfigMap, but for now, this will be helpful.

Perhaps you should make it clear that the user will need to edit the downloaded cert-manager manifests file, before applying it.